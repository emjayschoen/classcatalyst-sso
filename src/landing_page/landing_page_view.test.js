import React from 'react';
import { shallow } from 'enzyme';
import LandingPageView from './landing_page_view';

describe('<LandingPageView />', () => {
  let subject;
  beforeEach(() => {
    subject = shallow(<LandingPageView />);
  });

  it('renders correctly', () => {
    expect(subject).toMatchSnapshot();
  });
});
