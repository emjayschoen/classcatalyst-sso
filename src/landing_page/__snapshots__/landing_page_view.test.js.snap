// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPageView /> renders correctly 1`] = `
<styled.div>
  <styled.div>
    <b>
      CHANGEME-title
    </b>
  </styled.div>
  <styled.div>
    <styled.div>
      <h1 />
      <h2 />
    </styled.div>
  </styled.div>
  <styled.div>
    <hr />
    <Connect(ReduxForm) />
  </styled.div>
  <styled.div>
    <styled.div>
      Â© Copyright CHANGEME-title 2018. All Rights Reserved.
    </styled.div>
  </styled.div>
</styled.div>
`;
